tests/unit_tests/test_launch/test_builder/test_build.py

```diff
@@ -7,6 +9,21 @@
 from wandb.sdk.launch.builder.abstract import registry_from_uri
 from wandb.sdk.launch.builder.context_manager import get_requirements_section
 from wandb.sdk.launch.builder.templates.dockerfile import PIP_TEMPLATE
+from wandb.sdk.launch.create_job import _configure_job_builder_for_partial
+
+
+def _read_wandb_job_json_from_artifact(artifact):
```

Comment:
nit: not always necessary in test helpers, but a type annotation would be relatively easy to add here. Looks like `artifact: wandb.Artifact` and the return type is `str`?